### DISCUSSION TITLE
Giving staff an indication before an email is sent to the applicant

### DISCUSSION THREAD

**Discussion by wes-otf** [OpenTechFund/opentech.fund#4035] [General]
Staff are frequently confused about when applicants are sent an email notification about a change/transition made to their application. A small warning on the `Update Status` prompt when a status is selected that will notify the applicant would be helpful. Maybe requiring some HTMX in that modal?

**Comment by frjo** [comment]
Why does it matter when a status update goes out? I can see why an organisation would like to turn it off globally but not why staff should be bother by it.
